contracts/smart-account/interfaces/modules/SessionKeyManagers/ISessionKeyManagerModuleHybrid.sol

```diff
@@ -0,0 +1,66 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.20;
+
+/* solhint-disable no-empty-blocks*/
```

Comment:
this rule is necessary here? as there are no empty blocks

Reply:
removed